test/state/host.cpp

```diff
@@ -266,6 +266,17 @@ evmc::Result Host::execute_message(const evmc_message& msg) noexcept
 
     // Copy of the code. Revert will invalidate the account.
     const auto code = dst_acc != nullptr ? dst_acc->code : bytes{};
+
+    if (msg.kind == EVMC_DELEGATECALL)
+    {
+        // TODO: does the sender always have code here? or is this function used for EOAs too?
```

Comment:
used for EOAs too, as I understand

Reply:
Obsolete as the implementation has been moved to call instructions.